app/views/lois/new.html.erb

```diff
@@ -30,17 +30,90 @@
+                  </tr>
+            <% elsif section.name == "Geographic Focus" %>
+                <table class="table">
+                  <tr>
+                    <th>Focus</th>
+                    <th>Please select whichever applies (if more than one, pick one based on your main program)</th>
+                  </tr>
+            <% end %>
+
             <% @questions.each do |question| %>
-              <% if question.section_id == section.id %>
-                <div class="input-group">
+
+              <% if question.section_id == section.id && section.name == "Organizational Capacity Challenges" %>
+                
+                  <tr>
+                    <td><%= label_tag question.question+':' %></td>
+
+                    <td>
+                      <div class="challenge"><%= text_field_tag "answers[#{question.id}[challenges][]", nil, class: "form-control" %><p>Add</p></div>
+                    </td>
+                    <td>
+                      <div class="priority"><%= select_tag "answers[#{question.id}[challenges][]", options_for_select(["H", "M", "L"], "H") %></div>
+                    </td>
+
+                    <td><%= text_field_tag "answers[#{question.id}[ftes][]", nil, class: "form-control" %></td>
+                    <td><%= text_field_tag "answers[#{question.id}[ftes][]", nil, class: "form-control" %></td>
+                    <td><%= text_field_tag "answers[#{question.id}[ftes][]", nil, class: "form-control" %></td>
+                  </tr>
+
+                <% elsif question.section_id == section.id && section.name == "Demographics" %>
+
+                  <tr>
+                    <td><%= label_tag question.question+':' %></td>
+                    <td><%= check_box_tag "answers[#{question.id}]", "yes", false %></td>
+                  </tr>
+
+                <% elsif question.section_id == section.id && section.name == "Geographic Focus" %>
+
+                  <tr>
+                    <td><%= label_tag question.question+':' %></td>
+                    <td><%= check_box_tag "answers[#{question.id}]", "yes", false %></td>
+                  </tr>
+
+                <% elsif question.section_id == section.id %>
+
                   <li><%= label_tag question.question+':' %>
                   <%= text_field_tag "answers[#{question.id}]", nil, class: "form-control" %></li>
-                </div>
+                  
+                <% end %>
               <% end %>
-            <% end %>
+
+              <% if section.name == "Organizational Capacity Challenges" %>
+                <tr>
+                  <td></td>
+                  <td></td>
+                  <td>TOTAL FTEs</td>
+                  <td>2015 total</td>
+                  <td>2016 total</td>
+                  <td>2017 total</td>
+                </tr>
+              <% elsif section.name == "Demographics" %>
+                </table>
+              <% elsif section.name == "Geographic Focus" %>
```

Comment:
No need to fix now - but for future, remember the || operator. This could be
`<% elsif section.name == "Demographics || Geographic Focus" %>`. Any time you can cut down on the number of `elsif` that's a win.